addon/mixins/regenerated/models/new-platform-flexberry-g-i-s-map.js

```diff
@@ -32,66 +58,80 @@ export let Model = Ember.Mixin.create({
+    creator: Projection.attr('Создатель'),
+    createTime: Projection.attr('Время создания'),
+    editor: Projection.attr('Редактор'),
+    editTime: Projection.attr('Время редактирования')
   });
-  model.defineProjection('MapE', 'new-platform-flexberry-g-i-s-map', {
-    name: Projection.attr('Name'),
-    lat: Projection.attr('Lat'),
-    lng: Projection.attr('Lng'),
-    zoom: Projection.attr('Zoom'),
-    public: Projection.attr('Public'),
-    coordinateReferenceSystem: Projection.attr('CRS'),
+
+  modelClass.defineProjection('Map', 'new-platform-flexberry-g-i-s-map', {
+    name: Projection.attr('Наименование'),
+    lat: Projection.attr('Широта', { hidden: true }),
+    lng: Projection.attr('Долгота', { hidden: true }),
+    zoom: Projection.attr('Зум', { hidden: true }),
+    public: Projection.attr('Общая', { hidden: true }),
+    coordinateReferenceSystem: Projection.attr('Система координат', { hidden: true })
+  });
+
+  modelClass.defineProjection('MapE', 'new-platform-flexberry-g-i-s-map', {
+    name: Projection.attr('Наименование'),
+    description: Projection.attr('Описание'),
+    keyWords: Projection.attr('Ключевые слова'),
+    lat: Projection.attr('Широта'),
+    lng: Projection.attr('Долгота'),
+    zoom: Projection.attr('Зум'),
+    public: Projection.attr('Общая'),
+    scale: Projection.attr('Масштаб'),
+    coordinateReferenceSystem: Projection.attr('Система координат'),
+
+    // TODO: Uncomment when ODataService will support 'geography' type.
+    //boundingBox: Projection.attr('Граница'),
     mapLayer: Projection.hasMany('new-platform-flexberry-g-i-s-map-layer', '', {
-      name: Projection.attr('Name'),
-      type: Projection.attr('Type'),
-      visibility: Projection.attr('Visibility'),
-      index: Projection.attr('Index'),
-      coordinateReferenceSystem: Projection.attr('CRS'),
-      settings: Projection.attr('Settings'),
-      parent: Projection.belongsTo('new-platform-flexberry-g-i-s-map-layer', 'Parent', {
-
-      }),
-      map: Projection.belongsTo('new-platform-flexberry-g-i-s-map', 'Map', {
-
-      }),
+      name: Projection.attr('Наименование'),
+      description: Projection.attr('Описание'),
+      keyWords: Projection.attr('Ключевые слова'),
+      index: Projection.attr('Индекс'),
+      visibility: Projection.attr('Видимость'),
+      type: Projection.attr('Тип'),
+      settings: Projection.attr('Настройки'),
+      scale: Projection.attr('Масштаб'),
+      coordinateReferenceSystem: Projection.attr('Система координат'),
+
+      // TODO: Uncomment when ODataService will support 'geography' type.
+      //boundingBox: Projection.attr('Граница'),
+      parent: Projection.belongsTo('new-platform-flexberry-g-i-s-map-layer', 'Родитель', {
+
+      }, { hidden: true }),
+      map: Projection.belongsTo('new-platform-flexberry-g-i-s-map', 'Карта', {
+
+      }, { hidden: true }),
       layerLink: Projection.hasMany('new-platform-flexberry-g-i-s-layer-link', '', {
```

Comment:
this projection should stay the same. I believe that flexberry generator can not scaffold such projections. If you use the generated one, query and goto's to edit and list forms will not work as I remember. Try to test it

Reply:
Fixed.